uint8 STATUS_A = 0
uint8 STATUS_B = 1
uint8 STATUS_C = 2
uint8 STATUS_D = 3
uint8 STATUS_DEFAULT = 99

float32 PI = 3.14159265358979
float32 D2R = 0.01745329251   # PI/180.0
float32 R2D = 57.2957795131   # 180.0/PI
